### PR TITLE
Update create-retention-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information about retention limitations for private channels

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4851

File: microsoft-365/compliance/create-retention-policies.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ When you have more than one retention policy, and when you also use retention la
 
 3. For the **Choose locations to apply the policy** page, select one or both of the locations for Teams: **Teams channel message** and **Teams chats**.
 
-   For **Teams channel messages**, message from standard channels but not [private channels](/microsoftteams/private-channels) are included. Currently, private channels aren't supported by retention policies.
+   For **Teams channel messages**, message from standard channels are included, but not [private channels](/microsoftteams/private-channels). Teams messages in private channels are currently not supported for retention policies. Code snippets, recorded voice memos from the Teams mobile client, thumbnails, announcement images, and reactions from others in the form of emoticons are not retained when you use retention policies for Teams. For more information, See [Teams Retention policies](retention-policies-teams.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-retention-policies.md]:
```suggestion
   For **Teams channel messages**, message from standard channels are included, but not [private channels](/microsoftteams/private-channels). Teams messages in private channels are currently not supported for retention policies. Code snippets, recorded voice memos from the Teams mobile client, thumbnails, announcement images, and reactions from others in the form of emoticons are not retained when you use retention policies for Teams. For more information, see [Teams Retention policies](retention-policies-teams.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cabailey** [comment]
#hold-off

**Comment by cabailey** [comment]
Closing.  We already have the statement that private channels aren't supported (in our experience, customers are aware of this limitation) and a link to where we publish the authoritative source for what's included and excluded for Teams retention messages.  We want to avoid duplicating technical details like these so that when things change, the support statements don't get out of sync and people get confused which is correct.

**Pull request closed without merging**